### DISCUSSION TITLE
Predefined placeholders

### DISCUSSION THREAD

**Discussion by ghost** [jgraph/drawio-desktop#1349] [Q&A]
Hi,
 I try to use the predefined placeholders %page% to display on my page the name.
 What is the best method to do that.  Actually that does not work for me.

Christian

**Comment by tmal-1** [comment]
I imagine you see '%page%' instead of the actual name of the page.

Right click the object and select Edit > Edit Data ...
In the dialogue that appears there should be a checkbox, 'Placeholders'.
Make sure that is selected then click 'Apply'.
